cpp/test/data/verification_problems.info

```diff
@@ -0,0 +1,37 @@
+device {
+    #include "super_capacitor.info"
+}
+
+verification_problem_subramanian {
+    terms_in_truncation_of_infinite_series 10000
+
+    charge_current        5.0e-3 ;  [amperes]
+    charge_time           2.0    ;  [seconds]
+    time_step             0.01   ;  [seconds]
+
+    percent_tolerance     0.1    ;  [percent]
+}
+
+verification_problem_srinivasan {
+    terms_in_truncation_of_infinite_series 10000
+
+    initial_voltage       2.5    ; [volts]
+    discharge_current     5.0e-3 ; [amperes]
+    discharge_time        2.0    ; [seconds]
+    time_step             0.01   ; [seconds]
+
+    percent_tolerance     0.1    ; [percent]
```

Comment:
Correct.
BOOST_CHECK_CLOSE uses percent tolerance.
0.1% here is equivalent to a 0.001 relative tolerance.